.github/ISSUE_TEMPLATE/update_truth.md

```diff
@@ -26,9 +26,9 @@ assignees: ''
 - [ ] Select **engineer(s)** or **no engineer** required
 - [ ] Select **scientist(s)** or **no scientist** required
 
-### Projects and Milestone ###
-- [ ] Select **Repository** and/or **Organization** level **Project(s)** or add **alert: NEED CYCLE ASSIGNMENT** label
-- [ ] Select **Milestone** as the next official version or **Future Versions**
+### Milestone and Projects ###
+- [ ] Select **Milestone** as the next official version or **Backlog of Development Ideas**
```

Comment:
Updating the truth data should never be added to the backlog since it should be completed ASAP. If updating truth for an existing release, the bugfix milestone should be used instead.
```suggestion
- [ ] Select **Milestone** as the next official version if updating truth data for the `develop` branch OR select next X.Y.Z bugfix version if updating truth data for a `main_vX.Y` branch
```

Reply:
@georgemccabe Great point!  Clearly I wasn't using my brain for this one!🤪